src/main/java/teammates/common/datatransfer/FeedbackQuestionDetails.java

```diff
@@ -181,9 +176,12 @@ public String getNoResponseTextInHtml(String giverEmail, String recipientEmail,
                + Sanitizer.sanitizeForHtml(getNoResponseText(giverEmail, recipientEmail, bundle, question))
                + "</i>";
     }
-
-    public boolean shouldShowNoResponseText(String giverEmail, String recipientEmail,
-                                            FeedbackQuestionAttributes question) {
+    
+    /**
+     * returns true if 'No Response' to be displayed in the Response rows
+     * Used to construct missing response rows between the recipient and possible givers.
```

Comment:
- Methods should be independent of its calling site, so don't mention anything about "Used to ..."
- Convert the `returns true if ...` to `@return true if ...`. Also, "... if 'No Response' *is* to be ...".